chapters/ch28.md

```diff
@@ -1,51 +1,69 @@
 # 28. Diagnosing bias and variance: Learning curves
 ->
 
+# 28. Chuẩn đoán độ chệch và phương sai: Đồ thị quá trình học
+
 We’ve seen some ways to estimate how much error can be attributed to avoidable bias vs. variance. We did so by estimating the optimal error rate and computing the algorithm’s training set and dev set errors. Let’s discuss a technique that is even more informative: plotting a learning curve.
```

Comment:
Câu "We’ve seen some ways to estimate how much error can be attributed to avoidable bias vs. variance" hơi khó dịch. Ở đây mình dịch như dưới đây nhưng cảm thấy chưa ổn lắm :
* "Chúng ta đã biết một số cách để ước tính tỉ lệ lỗi tạo ra bởi độ chệnh tránh được so với phương sai."

Mọi người

Reply:
Mình thấy có thể dịch: "Chúng ta đã thấy một số cách để ước tính có bao nhiêu lỗi tạo ra bởi độ chệnh tránh được so với phương sai."